File: pension-management-portal/src/app/model/pensioner.ts
import {BankDetails} from "./bank-details";

export interface Pensioner {
  aadharNumber: number,
  name: string,
  dateOfBirth: string,
  pan: string,
  salaryEarned: number,
  allowances: number,
  pensionType: string,
  bankDetails: BankDetails
}

File: process-pension/src/main/java/dev/shuktika/processpension/model/Pensioner.java
package dev.shuktika.processpension.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;


@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Pensioner {
    private Long aadharNumber;
    private String name;
    private LocalDate dateOfBirth;
    private String pan;
    private Integer salaryEarned;
    private Integer allowances;
    private String pensionType;
    private BankDetails bankDetails;
}

